src/Symfony/Component/Config/Tests/Builder/Fixtures/PrimitiveTypes/Symfony/Config/PrimitiveTypesConfig.php

```diff
@@ -87,30 +107,41 @@ public function __construct(array $value = [])
     {
     
         if (isset($value['boolean_node'])) {
+            $this->_usedProperties['booleanNode'] = true;
             $this->booleanNode = $value['boolean_node'];
             unset($value['boolean_node']);
         }
     
         if (isset($value['enum_node'])) {
+            $this->_usedProperties['enumNode'] = true;
             $this->enumNode = $value['enum_node'];
             unset($value['enum_node']);
         }
     
         if (isset($value['float_node'])) {
+            $this->_usedProperties['floatNode'] = true;
             $this->floatNode = $value['float_node'];
             unset($value['float_node']);
         }
     
         if (isset($value['integer_node'])) {
+            $this->_usedProperties['integerNode'] = true;
             $this->integerNode = $value['integer_node'];
             unset($value['integer_node']);
         }
     
         if (isset($value['scalar_node'])) {
+            $this->_usedProperties['scalarNode'] = true;
             $this->scalarNode = $value['scalar_node'];
             unset($value['scalar_node']);
         }
     
+        if (isset($value['scalar_node_with_default'])) {
```

Comment:
shouldn't this use `array_key_exists` to allow passing `null` ?

Reply:
@stof Yep, missed that one, tnx. Fixed https://github.com/symfony/symfony/compare/e214761c573ecc980b5d1b5e3c4dff12f8a6933d..1264225f4ebc535deec491737b70d9bc1d84dc98